lib/sorcery/controller.rb

```diff
@@ -1,4 +1,6 @@
 module Sorcery
+  class InvalidCredentials < StandardError; end
```

Comment:
Not sure if it's the best place to define this exception?

Reply:
Here's what I did in the v1 codebase:

https://github.com/Sorcery/sorcery-rework/blob/main/sorcery-core/lib/sorcery/errors.rb

If you follow a similar pattern, it should be easy to migrate this functionality into the new codebase, once I finally can sit down and finish it.

Reply:
Done in 0592c84